grafana-plugin/src/models/user/user.ts

```diff
@@ -52,6 +52,7 @@ export class UserStore {
           (acc: { [key: number]: ApiSchemas['User'] }, item: ApiSchemas['User']) => ({
             ...acc,
             [item.pk]: {
+              ...this.items[item.pk],
```

Comment:
what does this change do?

Reply:
if some user comes from `GET /users` endpoint then it will be **merged** with the existing user **with the same id** that comes from `GET /user` endpoint (to avoid `google_calendar_settings` object disappearing)